src/index.js

```diff
@@ -1,21 +1,20 @@
 import { createStore, applyMiddleware } from 'redux';
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 import React from 'react';
+import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom';
 import App from './App';
+import reducers from './Reducer/index';
 import registerServiceWorker from './registerServiceWorker';
-import storeSite from './state';
 
-const customMiddleWare = store => next => action => {
```

Comment:
Also, could you please extract `isUrl` and `httpGet` helper function to, for example, `fetchHelpers.js`?